src/galaxy/db_connection.py

```
@@ -0,0 +1,82 @@
+# Copyright (C) 2021 Humanitarian OpenStreetmap Team
+
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+
+# You should have received a copy of the GNU Affero General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+# Humanitarian OpenStreetmap Team
+# 1100 13th Street NW Suite 800 Washington, D.C. 20005
+# <info@hotosm.org>
+
+from psycopg2 import pool
+from .config import logger as logging
+from .config import get_db_connection_params
+
+
+class Database:
+    """Handles the all work related to connection pooling
+    """
+    def __init__(self):
+        self.db_params = get_db_connection_params("RAW_DATA")
+        self._cursor = None
+        self.threaded_postgresql_pool  = None
+        self.con = None
+
+    def connect(self):
+        """Connection to the database 
+        """
+        if not self.threaded_postgresql_pool :
+            try:
+                #creating pool through psycopg2 with threaded connection so that there will support from 3 users to 20 users 
+                self.threaded_postgresql_pool = pool.ThreadedConnectionPool(3, 20,**self.db_params)
+                logging.debug("Connection pooling has been established")
+                if (self.threaded_postgresql_pool):
+                    logging.info("Connection pool created successfully using ThreadedConnectionPool")
+            except Exception as ex:
+                logging.error(ex)
```

`raise` instead of `logging` ?